docs/source/examples.rst

```diff
@@ -0,0 +1,208 @@
+Examples
+=========
+
+Tumour Pair Full Pipeline
+----------------------------
+
+The following example is a tumour normal genome pair with a tumour transcriptome. All of which was set up to run on :term:`SLURM`.
```

Comment:
The second sentence is a fragment. Should be a comma separating the two sentences rather than a period